src/utils.js

```diff
@@ -184,7 +184,7 @@ function getFillerOffset() {
 //
 // @param {module:engine/view/editableelement~EditableElement}
 // @param {module:engine/view/writer~Writer} writer
-function addSelectionHandler( editable, writer ) {
+function hasSelectionHandler( editable, writer ) {
```

Comment:
shouldn't this stay as `addSelectionHandler()` method?

Reply:
My bad. Search/replace. Fixing :/